<launch>
  <arg name="graph_file" default="/home/nvidia/catkin_ws/src/lab4/graph_files/cse_floor_four.graphml"/>
  <arg name="visualize" default="true"/>
  <node pkg="lab4" type="PlannerNode.py" name="planner_node" output="screen">
    <param name="graph_file" value="$(arg graph_file)"/>
    <param name="visualize" type="bool" value="$(arg visualize)"/>
  </node>
</launch>
